starport/pkg/chaincmd/chaincmd.go

```diff
@@ -86,14 +101,22 @@ func WithKeyringBackend(keyringBackend KeyringBackend) Option {
 	}
 }
 
-// WithLaunchpad defines the command as Launchpad application commands
-// and provides the name of the CLI application to call Launchpad CLI commands
-func WithLaunchpad(cliCmd string) Option {
+// WithSecondaryCLI provides the secondary CLI name for the blockchain.
+// this is useful for Launchpad applications since it has two different binaries but
+// not needed by Stargate applications.
+func WithSecondaryCLI(cliCmd string) Option {
```

Comment:
I prefer to keep an explicit name that we are using `Launchpad` personally

Reply:
Let me address this in another PR. :+1: